treeherder/intermittents_commenter/commenter.py

```diff
@@ -169,6 +169,10 @@ def print_or_submit_changes(self, all_bug_changes):
                 # sleep between comment submissions to avoid overwhelming servers
                 time.sleep(1)
 
+        if settings.COMMENTER_API_KEY is None:
+            # confirm number of comments when on stage/dev
+            logger.info('There were {} comments for this {} task.'.format(len(all_bug_changes), 'weekly' if self.weekly_mode else 'daily'))
```

Comment:
At the moment production logging is set to warning level or higher:
https://github.com/mozilla/treeherder/blob/ae0f12f5ba9540f7998d3fcfbb52f956ca06dbae/treeherder/config/settings.py#L212

...so whilst this isn't a warning, we'll need to use `logger.warn()` for this to show up.

This line is also quite long (139 characters) - it might be worth adjusting the wrapping or else assigning `'weekly' if self.weekly_mode else 'daily'` to an intermediate variable.